apps/example-app/src/app/home/home.page.html

```diff
@@ -1,29 +1,48 @@
-<section>
-    <section class="sliders">
-        <div class="sliders-wrappers">
-            <div class="sliders-zone">
-                <p><ion-icon name="information-circle-outline" (click)="presentTooltip($event, 'Document')"></ion-icon> Document Module</p>
-                <ion-range min="10" max="20" step="1" value="16" (ionChange)="documentModuleValueChange($event.detail.value)"> </ion-range>
-                <p><ion-icon name="information-circle-outline" (click)="presentTooltip($event, 'Header')"></ion-icon> Header Module</p>
-                <ion-range min="0.6" max="1.2" step="0.2" value="1.2" (ionChange)="headerModuleValueChange($event.detail.value)"> </ion-range>
-                <p><ion-icon name="information-circle-outline" (click)="presentTooltip($event, 'Content')"></ion-icon> Content Module</p>
-                <ion-range min="0.6" max="2.0" step="0.2" value="1" (ionChange)="sectionModuleValueChange($event.detail.value)"> </ion-range>
-                <p><ion-icon name="information-circle-outline" value="'Footer Module Modifier'" (click)="presentTooltip($event, 'Footer')"></ion-icon> Footer Module</p>
-                <ion-range min="0.5" max="1.0" step="0.05" value="0.75" (ionChange)="footerModuleValueChange($event.detail.value)"> </ion-range>
-                <ion-button id="click-trigger" (click)="presentPopover($event)">
-                    {{ devicePhoneSelected }}
-                </ion-button>
-            </div>
-        </div>
-    </section>
-    <!-- DISPLAY THE PHONE AND PHONE CONTENT-->
-    <openforge-default-phone-layout *ngIf="devicePhoneSelected === 'Default Phone'">
-        <openforge-app-content></openforge-app-content>
-    </openforge-default-phone-layout>
-    <openforge-iphone-layout *ngIf="devicePhoneSelected === 'iPhone 6'">
-        <openforge-app-content></openforge-app-content>
-    </openforge-iphone-layout>
-    <openforge-iphone-x-layout *ngIf="devicePhoneSelected === 'iPhone X'">
-        <openforge-app-content></openforge-app-content>
-    </openforge-iphone-x-layout>
-</section>
+<div class="flex-container">
+    <div class="left-column">
+        <p>
+            <ion-icon name="information-circle-outline" (click)="presentTooltip($event, 'Document')"></ion-icon>
+            Document Root<br />
+            <i>Font-size: {{ rootFontSize }}px</i>
+            <ion-range min="10" max="20" step="1" value="16" (ionChange)="documentModuleValueChange($event.detail.value)"> </ion-range>
+        </p>
+
+        <p>
+            <ion-icon name="information-circle-outline" (click)="presentTooltip($event, 'Header')"></ion-icon>
+            Header Module: <br />
+            <i>{{ headerRem | number: '1.2-2' }}rem = {{ headerRem * rootFontSize | number: '1.2-2' }}px</i>
```

Comment:
Added this as a way to easily show the user what the root pixel will be and how the rem affects that.